lib/moonshot/cli.rb

```diff
@@ -47,6 +47,21 @@ def inherited(base)
         base.include(Moonshot::BuildMechanism)
         base.include(Moonshot::DeploymentMechanism)
       end
+
+      def ssh_options!
+        option :user, default: ENV['MOONSHOT_SSH_USER'] || ENV['USER'], type: :string,
+                      aliases: '-l',
+                      desc: 'Specifies the user to log in as on the remote machine.'
+        option :'identity-file', default: ENV['MOONSHOT_SSH_KEY_FILE'], type: :string,
```

Comment:
If you use `:identity_file`, Thor will handle both `--identity_file` and `--identity-file` correctly, and document the hyphenated form in `--help`.